config/event-emitter.php

```diff
@@ -1,6 +1,11 @@
 <?php
 
 return [
+    /**
+     * Event emitter source name to be used when logging
+     */
+    'source' => env('EVENT_EMITTER_ENABLE', 'undefined'),
```

Comment:
@carroaung we can use the `APP_NAME` env for default value if somewhere we forget to add it
And I think the name should be `EVENT_EMITTER_SOURCE`

```suggestion
    'source' => env('EVENT_EMITTER_SOURCE', env('APP_NAME', 'undefined')),
```

Reply:
updated 

https://github.com/carro-public/event-emitter/pull/4/commits/5cd96e18d3ef4834798e9f758100621a7a935cab